projects/transformers/export_model.py

```diff
@@ -31,13 +31,16 @@
 
 from transformers import AutoModelForMaskedLM
 
+# Import models. This will update Transformer's model mappings so that custom models can
+# be loaded via AutoModelForMaskedLM.
+import models # noqa
```

Comment:
I think the rule you are looking for here is F401, so # noqa: F401.

For completeness, a different way of doing this without the noqa: https://stackoverflow.com/questions/59167405/flake8-ignore-only-f401-rule-in-entire-file

Reply:
Added it.